lib/hanami/commands/generate/action.rb

```diff
@@ -234,14 +234,16 @@ def spec_root
         end
 
         def relative_action_path
-          path = Pathname.new('.').join('..', '..', '..')
-          path = path.join('..') if environment.container?
+          nestings = ['..'] * @controller_pathname.count('/')
```

Comment:
@berfarah We should replace hardcoded file separator `'/'` with `::File::SEPARATOR`.